src/test/java/co/rsk/bitcoinj/script/ScriptTest.java

```diff
@@ -753,9 +822,36 @@ public void isSentToMultiSig_whenFlyoverNonStandardErpRedeemScript_shouldReturnT
     }
 
     @Test
-    public void isStandardMultiSig_whenStandardRedeemScript_shouldReturnTrue() {
-        Script redeemScript = RedeemScriptUtils.createStandardRedeemScript(FEDERATION_KEYS);
-        Assert.assertTrue(redeemScript.isSentToMultiSig());
+    public void isSentToMultiSig_whenFlyoverP2shErpRedeemScript_shouldReturnTrue() {
+        Script redeemScript = RedeemScriptUtils.createP2shErpRedeemScript(
+            FEDERATION_KEYS,
+            ERP_FEDERATION_KEYS,
+            CSV_VALUE
+        );
+
+        Script flyoverRedeemScript = RedeemScriptUtils.createFlyoverRedeemScript(
+            FLYOVER_DERIVATION_HASH,
+            redeemScript
+        );
+
+        Assert.assertTrue(flyoverRedeemScript.isSentToMultiSig());
+    }
+
+    @Test
+    public void isSentToMultiSig_whenZeroHashFlyoverPrefix_shouldReturnTrue() {
```

Comment:
mmm whats the point of this test? We should never have a flyover rs with the zero hash

Reply:
This kind of demonstration test that shows that we are not checking the content of the flyover prefix here. We are delegating this to the flyover builder in rskj. However, I will add this information to the test in a comment.

Reply:
Comment added. Anyways, let's follow this discussion with the team to decide what's best. Thanks for notice it.